draft-ietf-gnap-core-protocol.md

```diff
@@ -1394,8 +1394,8 @@ sub_ids (array of objects)
 : An array of subject identifiers for the
             end user, as defined by {{I-D.ietf-secevent-subject-identifiers}}.
 
-assertions (object)
-: An object containing assertions as values keyed on the assertion
+assertions (array of objects/strings)
```

Comment:
```suggestion
assertions (array of objects)
```

I don't see anything here about string-formatted assertions, so should this just be objects?

Reply:
ok